### DISCUSSION TITLE
Suggestion: new vehicle color presets

### DISCUSSION THREAD

**Discussion by SpartanFrederic104** [OpenRCT2/OpenRCT2#20970] [Feature requests]
With #19446 adding new colors to the color dropdown menu and both #20119 and #20243 adding new track color presets to several roller coaster types, a similar idea crossed my mind: what if the same idea was implemented for ride vehicles?

For instance, the floorless trains could receive a new color preset of bright yellow, bright purple, and bright orange, which is based on the train color scheme of [Dominator](https://rcdb.com/4079.htm), a B&M floorless coaster at Kings Dominion.
![Fred's Workbench 2023-11-11 18-26-28](https://github.com/OpenRCT2/OpenRCT2/assets/61885939/5e1542cd-3c3c-494e-a69e-b6d8eca04a5e)

If you have any ideas for new vehicle color presets, list them down below.

**Comment by ZehMatt** [comment]
I suggested this before in a previous PR, ideally users can create and load presets in the Options window, that seems like the way to make everyone happy. I don't think it should be tied to rides or tracks, there should be just one table of presets.

**Reply by SpartanFrederic104** [reply]
I like that idea better actually.

**Reply by ocalhoun6** [reply]
>I don't think it should be tied to rides or tracks

That part, I'd disagree with, though.

Because certain color schemes might work well for some types of rides, but look terrible on others. And it would overall make for a much more boring park-wide color scheme if _every_ new ride was pulling from the same set of color presets.

For example, the wooden roller coaster usually looks best with supports being white or various shades of brown ... but you don't want to limit steel coaster types to only having white or brown supports.

_Maybe_ ride vehicles could all pull from the same collection of color schemes, but you definitely _do_ need to tie it to tracks, at least.
